test/test_server_selection_in_window.py

```diff
@@ -127,10 +127,13 @@ def frequencies(self, client, listener):
     @client_context.require_multiple_mongoses
     def test_load_balancing(self):
         listener = OvertCommandListener()
+        # Use a large localThresholdMS to avoid the impact of varying RTTs.
```

Comment:
Minor nit: can we add the ticket number here? The ticket description has a great summary of what the exact problem is. Having it in the comment is a bit friendlier than need to check the blame in my book...

Reply:
Done.